luke/index.js

```diff
@@ -0,0 +1,41 @@
+// task 1
+let snackServings = 26;
+let guests = 9;
+
```

Comment:
You forgot Add snackServings and guests, and print the result to the console
console.log(snackServings + guests) could be the solution!

Reply:
sorry i must have not read that properly